src/os/bluestore/KernelDevice.cc

```diff
@@ -855,7 +855,8 @@ int KernelDevice::direct_read_unaligned(uint64_t off, uint64_t len, char *buf)
 {
   uint64_t aligned_off = align_down(off, block_size);
   uint64_t aligned_len = align_up(off+len, block_size) - aligned_off;
-  bufferptr p = buffer::create_small_page_aligned(aligned_len);
+  auto& p = buffer::ptr_node::create(
```

Comment:
The `p` isn't destined for a bufferlist. Old `bufferptr` was enough then. Switching back.

Reply:
FIXED.